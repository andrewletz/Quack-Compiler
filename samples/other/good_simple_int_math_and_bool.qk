/**
 * Verifies sugaring of integer arithmetic.
 * Also verifies printing for Booleans, Nothing, and Integers.
 */
(5 + 4).PRINT(); """\n""".PRINT();
(95 / 5).PRINT(); """\n""".PRINT();
(5 * 6).PRINT(); """\n""".PRINT();
(3 - 20).PRINT(); """\n""".PRINT();

true.PRINT(); """\n""".PRINT();
false.PRINT(); """\n""".PRINT();

none.PRINT(); """\n""".PRINT();
