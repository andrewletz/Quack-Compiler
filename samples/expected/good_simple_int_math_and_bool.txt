9
19
30
-17
true
false
<nothing>
